src/Workspaces/Remote/ServiceHub/Services/BrokeredServiceBase.cs

```diff
@@ -76,8 +76,8 @@ public virtual void Dispose()
         public RemoteWorkspace GetWorkspace()
             => WorkspaceManager.GetWorkspace();
 
-        public HostWorkspaceServices GetWorkspaceServices()
-            => GetWorkspace().Services;
+        public SolutionServices GetWorkspaceServices()
```

Comment:
> GetWorkspaceServices

Consider renaming to `GetSolutionServices`?

Reply:
I'm primarily concerned with using the right types and not leaking out stuff that isn't necessary.  I'd prefer to keep these PRs minimal and not clutter with naming that doesn't substantially improve readability to me.  thanks!